e2e/cypress/tests/integration/accessibility/accessibility_sidebar_spec.ts

```diff
@@ -43,15 +43,15 @@ describe('Verify Accessibility Support in Channel Sidebar Navigation', () => {
         }).then(() => {
             // # Post messages to have unread messages to test user
             for (let index = 0; index < 5; index++) {
-                cy.postMessageAs({sender: otherUser, message: 'This is an old message', channelId: otherChannel.id});
+                (cy as any).postMessageAs({sender: otherUser, message: 'This is an old message', channelId: otherChannel.id});
```

Comment:
This command was migrated in this PR, so we should be able to avoid this `any` if we add the type to `postMessageAs`. This type seemed to work on my [e2e-ts](e2e/cypress/tests/integration/accessibility/accessibility_sidebar_spec.ts) branch:

```ts
interface PostMessageArg {
    sender: {
        username: string;
        password: string;
    };
    message: string;
    channelId: string;
    rootId?: string;
    createAt?: number;
}
function postMessageAs(arg: PostMessageArg): ChainableT<PostMessageResponse> {
    const {sender, message, channelId, rootId, createAt} = arg;
```